modules/nlu/src/backend/predict-pipeline.ts

```diff
@@ -159,7 +160,16 @@ async function extractEntities(input: PredictStep, predictors: Predictors, tools
   return { ...input }
 }
 
+const getCustomEntitiesNames = (predictors: Predictors): string[] => [
+  ...predictors.list_entities.map(e => e.entityName),
+  ...predictors.pattern_entities.map(e => e.name)
+]
+
 async function predictContext(input: PredictStep, predictors: Predictors): Promise<PredictStep> {
+  const customEntities = [
```

Comment:
Can't you use the function you've declared two lines above? (`getCustomEntitiesNames`)